SDWebImageSwiftUI/Classes/WebImage.swift

```diff
@@ -66,10 +66,12 @@ public class WebImagePlayer: ObservableObject {
     
     var selfId = UUID()
     
+    var isCacheImage = false
     var imageManager: ImageManager
+    
     @Published var currentFrame: PlatformImage?
     var imagePlayer: SDAnimatedImagePlayer?
-    
+    var cachingId: Set<String> = []
```

Comment:
Potentially re-name to `loadedPlayerViewIDs`

Reply:
done